CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+4.46.2
+==========
+
+### New Features
+- WAM Authentication Library Should Support .NET 4.5 for Back Compatibility. See [#3539](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3539)
```

Comment:
```suggestion
- WAM Authentication Library now explicitly supports .NET 4.6.2. See [#3539](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3539)
```

Reply:
(and yes, I know that it actually targets 4.6.1, but 4.6.1 is out of support. 4.6.2 is the only one still supported)